### DISCUSSION TITLE
Cannot install - error 'drive or UNC share' and then force closes

### DISCUSSION THREAD

**Discussion by Nearox** [tsunamods-codes/7th-Heaven#245] [Q&A] [answered]
Hi there!

I cannot seem to install the software. 

It gives me this error: 

![Image](https://github.com/user-attachments/assets/a64a911e-2086-4191-b7d6-1f16d65a7588)

I've tried running the installer from different drives, to no avail. Weird thing is, I don't even get to select a drive at all. It gives this error immediately after launching, after selecting the desktop shortcut... 

Is there a portable version available? Or would you be able to tell me how to fix this?

Thank you so much for your work!

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hello, usually for support we do prefer to be approached via Discord as it's faster to troubleshoot. However based on the error I see to be honest, it's nothing we can fix as it seems it's related to how your Windows installation has been made. Try to see if 7th Heaven is listed as already installed in the control panel, try to unistall it and then reinstall it again. If the setups detects it has already been installed, it doesn't allow you to pick another path as it's considered an in-place upgrade. Hope this helps, cheers!

**Comment by Nearox** [comment]
Thank you so much for your support, and thank you once again for a wonderful addon manager!

Your tip helped, apparently I had an old installation in the control panel. Removed that and it worked. Cheers :)
